app/views/layouts/application.html.erb

```diff
@@ -29,30 +29,32 @@
 
   <%= render "govuk_publishing_components/components/skip_link" %>
 
-  <%= render "govuk_publishing_components/components/cookie_banner", {
-    title: t("cookie_banner.title"),
-    text: t("cookie_banner.description"),
-    confirmation_message: t("cookie_banner.confirmation_message"),
-    cookie_preferences_href: t("cookie_banner.cookie_settings_href"),
-    services_cookies: {
-      yes: {
-        text: "Yes",
-        data_attributes: {
-          "track-category": "cookieBanner",
+  <% unless @skip_cookie_banner %>
```

Comment:
Is this executed before `your_information.html.erb` is parsed?  I'm just wondering when the variable will actually get set?

Reply:
Yes, the view gets executed before the layout